applications/GeoMechanicsApplication/geo_mechanics_application_constants.h

```diff
@@ -0,0 +1,87 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Vahid Galavi
+//
+
+#if !defined(KRATOS_GEO_MECHANICS_APPLICATION_CONSTANTS_H_INCLUDED )
+#define  KRATOS_GEO_MECHANICS_APPLICATION_CONSTANTS_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+
+
+namespace Kratos
+{
+    /* Contants used in GeoMechanicsApplication */
+    // The size type definition
+    typedef std::size_t         SizeType;
+
+
+    // Static definition of the dimension
+    constexpr SizeType N_DIM_3D = 3;
+    constexpr SizeType N_DIM_2D = 2;
+    constexpr SizeType N_DIM_1D = 1;
+
+    // Limits
+    constexpr double TINY  = 1.0e-60;
+    constexpr double LARGE = 1.0e10;
+
+    // factor for pore pressure calculations: 1: mechanical sign convention, -1: soil mechanics sign convention
+    constexpr double PORE_PRESSURE_SIGN_FACTOR = 1.0;
```

Comment:
what is the intended usecase for those?

Reply:
I want to use compiler independent limits for large and small numbers.
The pore_pressure_sign_factor is used to simply switch between soil mechanics and structural mechanics sign convention for water pressure (stress). By default,